lib/vechain/test/.env-test

```
@@ -0,0 +1,33 @@
+# General Settings
+AWS_ACCOUNT_ID="351534084702"                   # Replace with your AWS account ID
```

Let's remove this before we forget
```suggestion
AWS_ACCOUNT_ID="xxxxxxxxxxx"                   # Replace with your AWS account ID
```